src/ontology/mondo-edit.obo

```diff
@@ -369,7 +369,7 @@ subset: otar {source="MONDO:OTAR"}
 subset: rare
 synonym: "epilepsy, nocturnal frontal lobe, familial" EXACT [OMIMPS:600513]
 synonym: "familial sleep-related hyperkinetic epilepsy" EXACT [https://www.epilepsydiagnosis.org/syndrome/adnfle-overview.html, PMID:28027860]
-synonym: "familial sleep-related hypermotor epilepsy" EXACT CLINGEN_LABEL [PMID:27164717, PMID:28027860]
+synonym: "familial sleep-related hypermotor epilepsy" EXACT CLINGEN_LABEL [PMID:27164717, PMID:28027860] {OMO:0002001="https://www.wikidata.org/entity/Q64403342"}
```

Comment:
Would the tag `CLINGEN_LABEL` be removed in this line since, as I thought, that is the reason for the new modeling of the preferred labels.

Is there an infores solution that is more user friendly than this URL to Wikidata for ClinGen? It's machine parsable, but is there any need for curators to more clearly see this is the clingen label tag?

Reply:
Ok but just be aware that this is a single example which, if you merge this, wont have CLINGEN_LABEL as a synonym type anymore!